### PR TITLE
patch(116): update the fields to match vtm app

### LINKED ISSUE #116: Make all 'required' validation consistent
| type | Skeleton | Testable | Complete |
| -----| -------- | -------- | --------- |
| Car | missing vehicle configuration | NA | missing vehicle config
| Motorcycle |  OK | NA | vehicle configuration marked required in FE, missing here
| LGV | OK | NA | no of axles marked required in the frontend, missing here
| Small trailer |  OK | NA | techRecord_vehicleSubclass can be null
| HGV | vehicle class and body type required in FE, missing here | no of axles, body type required in FE, missing here |  |
| PSV |  body make and type, chassis make and model are not required on FE, but dtp fills in the fields | dtp number is required but missing | |
| Trailer | OK | OK| |

### PR DESCRIPTION
**Pull request by shivangidas**
## Make fields and validations consistent

Closes [116](https://github.com/dvsa/cvs-type-definitions/issues/116)

## Changelog

- update PUT schema to match frontend

### CODE REVIEW

File: json-definitions/v3/tech-record/put/car/complete/index.json [hunk 1]
```diff
@@ -71,6 +72,16 @@
         "type": "string"
       }
     },
+    "techRecord_vehicleConfiguration": {
```
Comment (gjulien-bjss) [inline json-definitions/v3/tech-record/put/car/complete/index.json:75]: I'm confused... how does the API save this value if it's not on the json schema? (seeing as it strips all unknown values)

Comment (shivangidas) [reply json-definitions/v3/tech-record/put/car/complete/index.json:75]: it doesn't save the value, the car config is defaulted to 'other' on frontend and we always see that, even if you change it

Comment (gjulien-bjss) [reply json-definitions/v3/tech-record/put/car/complete/index.json:75]: Ah. Yeah that's not good. Kinda feels like a bug that would need bringing into the release.

Comment (shivangidas) [reply json-definitions/v3/tech-record/put/car/complete/index.json:75]: I am not convinced vehicle config defaulted to other is a required field, seems like something we carried over from bad data

**Review by gjulien-bjss — APPROVED** [review]
I would just check the GET types match the rules on the fields here, since it doesn't look like any of those have been updated. Looks good otherwise

**Review by naathanbrown — APPROVED** [review]

**Review by kris-pearman — APPROVED** [review]

**Pull request merged**